js/contextMenus.js

```diff
@@ -683,7 +683,7 @@ function hamburgerTemplateInit (location, e) {
     }, {
       label: locale.translation('bravery'),
       submenu: [
-        CommonMenu.braveryGlobalMenuItem(),
+//        CommonMenu.braveryGlobalMenuItem(),
```

Comment:
nit: Comment should be aligned with the lines on the same indent level
nit2: You don't ever really need to comment out code since all code is recoverable with git history.